moveit_ros/planning_interface/robot_interface/src/wrap_python_robot_interface.cpp

```diff
@@ -223,16 +250,32 @@ class RobotInterfacePython : protected py_bindings_tools::ROScppInitializer
     return true;
   }
 
+  bool ensureNotMonitoring()
+  {
+    if (current_state_monitor_)
```

Comment:
Would also change this to `monitor_current_state` to avoid hard-to-catch errors in case `current_state_monitor_` is initialized to a falsy state.